src/utils/authMiddlewares.js

```diff
@@ -210,16 +212,33 @@ const authenticationMiddlewareSocketIO = async (authHeader) => {
  * and remove authByExperiment
  */
 const authorize = async (userId, resource, method, authResource, authByExperiment = true) => {
+  const isV2 = resource.startsWith('/v2/');
```

Comment:
I thought these files would be duplicated too instead of having to modify one to support both.

Reply:
so you are proposing moving this into each of the api's folders and making one for each version?

Reply:
I just hadn't noticed this. I feel that it's gonna become very messy if we migrate the whole code base like this because there are a lot of things in utils. What do you think?

Reply:
Are there any other things in utils that would require us to do this?